### ISSUE TITLE
AWX 21.5.0 edit workflow results in failure to save

### ISSUE THREAD

**Issue by nicolaibaralmueller** [ansible/awx#12956]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I might not receive a timely response.

### AWX version

21.5.0

### Select the relevant components

- [X] UI
- [ ] API
- [ ] Docs
- [ ] Collection
- [ ] CLI
- [ ] Other

### Installation method

kubernetes

### Modifications

no

### Ansible version

quay.io/ansible/awx-ee:latest

### Operating system

Ubuntu 22.04 LTS

### Steps to reproduce

Create workflow, save and then edit the template(s) containing survey, click save and a error message shows:

Error saving workflow!
There was an error saving the workflow.

```
PUT api/v2/workflow_job_template_nodes/9/ 400
This field cannot be empty.
```

Everything is filled out correctly. Only way to work around the issue it to delete and re-create the workflow with new changes.

### Expected results

Successful change in workflow templates with survey

### Actual results

See steps to reproduce

**Comment by AlekseiSaff** [comment]
https://github.com/ansible/awx/issues/12782

**Comment by nicolaibaralmueller** [comment]
Resolved with version 21.7.0
